app/src/main/java/com/example/cm/ui/meetup/MeetupDetailed/MeetupDetailedFragment.java

```diff
@@ -106,10 +95,53 @@ private void initUIAndViewModel() {
                     binding.meetupDetailedTime.setText(R.string.meetup_ended_text);
                     break;
             }
-
             initMenu(meetup);
+            initImg(meetup);
         });
+    }
+
+    private void initImg(Meetup meetup) {
+        Glide.with(requireActivity()).load(meetup.getLocationImageUrl()).placeholder(R.drawable.cafe)
+                .into(new CustomTarget<Drawable>() {
+            @Override
+            public void onResourceReady(@NonNull Drawable resource, @Nullable Transition<? super Drawable> transition) {
+                binding.ivLocation.setImageDrawable(resource);
+            }
 
+            @Override
+            public void onLoadCleared(@Nullable Drawable placeholder) {
+                binding.ivLocation.setImageDrawable(placeholder);
+            }
+        });
+    }
+
+    private void initTabbar() {
+        TabLayout tabLayout = binding.meetupDetailedTabLayout;
+
+        tabLayoutMediator = new TabLayoutMediator(tabLayout, viewPager, (tab, position) -> {
+            if (position == 0) {
+                tab.setText(R.string.meetup_tabs_label_accepted);
+            } else if (position == 1) {
+                tab.setText(R.string.meetup_tabs_label_declined);
+            } else if (position == 2) {
+                tab.setText(R.string.meetup_tabs_label_open);
+            } else if (position == 3) {
+                tab.setText(R.string.meetup_tabs_label_add);
+            }
+        });
+        tabLayoutMediator.attach();
+
+        ViewGroup slidingTabStrip = (ViewGroup)tabLayout.getChildAt(0);
+
+        View tab2 = slidingTabStrip.getChildAt(2);
```

Comment:
Maybe extract these Indices to a Constant file and name them something like "ACCEPTED_TAB_INDEX" etc.